### PR TITLE
[PWM/DV] Updates For DV document

### PR DESCRIPTION
**Pull request by dharanendrak**
Updated PWM DV document for V2
   PWM is ready for V2

Signed-off-by: Dharanendra Kumar <dharanendra.kumar@ensilica.com>

### CODE REVIEW

File: hw/ip/pwm/doc/dv/index.md [hunk 1]
```diff
@@ -138,21 +129,21 @@ The functional coverage plan can be found here: [coverageplan](#testplan)
...
+when a channel is configured to start sending pulses the first expected item is generated.
 Because of the way the PWM IP is design the first and the last pulse might not match the configuration settings.
 Therefore the scoreboard will wait until a channel is disabled before checking the output.
 Once a channel is disabled it will first discard the first two items received from the monitor.
 The is send because the channel was enabled and has no valid information.
 The second is the one that cannot be expected to match configuration.
 For pulse mode it will get the expected pulse item and match all incoming item to this one.
 For blink and heart beat mode after an item is compared successfully the scoreboard will generate the next expected item based on the previous item and the settings of the blink parameters.
-If an error is found the scoreboard will throw a fatal error.
+If an error is found the scoreboard will throw a error.
```
Comment (jdonjdon) [inline hw/ip/pwm/doc/dv/index.md:146]: throw *an error

**Review by jdonjdon — APPROVED** [review]
LGTM wit a nit

File: hw/ip/pwm/doc/dv/index.md [hunk 2]
```diff
@@ -138,21 +129,21 @@ The functional coverage plan can be found here: [coverageplan](#testplan)
 
 ### Self-checking strategy
 #### Scoreboard
-The `pwm_scoreboard` is primarily used for end to end checking.
+The `pwm_scoreboard` is primarily used for transaction-by-transaction checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
 * item_fifo[NUM_PWM_CHANNELS]: the FIFO w.r.t channels receives the dut items sent by the pwm_monitor
-* exp_item_q[PWM_NUM_CHANNELS]: the queues  w.r.t channels are used to store the expected/referenced items 
+* exp_item                   : It is a generated item used to store the expected/referenced items
```
Comment (matutem) [inline hw/ip/pwm/doc/dv/index.md:135]: This made more sense when talking about a queue, now the sentence makes little sense. Perhaps you meant just say "It is used to store the expected item constructed from tl address and data channels."?

**Review by matutem — APPROVED** [review]
Please handle the suggestion, otherwise LGTM.

File: hw/ip/pwm/doc/dv/index.md (hunk 1 above)
Comment (dharanendrak) [reply hw/ip/pwm/doc/dv/index.md]: I have updated the file with your review comment

File: hw/ip/pwm/doc/dv/index.md (hunk 2 above)
Comment (dharanendrak) [reply hw/ip/pwm/doc/dv/index.md]: I have updated the file with your review comment

**Comment by cindychip** [comment]
There seems to have a merge conflict. @dharanendrak could you kindly rebase to the latest and solve the conflict? Then I can merge it :)

**Review by cindychip — APPROVED** [review]

**Comment by cindychip** [comment]
sorry about that, I still see this in my screen:
image

have you tried following this instruction? https://docs.opentitan.org/doc/ug/github_notes/#dealing-with-conflicts-after-a-rebase

**Comment by dharanendrak** [comment]
> sorry about that, I still see this in my screen: image
> 
> have you tried following this instruction? https://docs.opentitan.org/doc/ug/github_notes/#dealing-with-conflicts-after-a-rebase

I did re-merge. Can you please check at your end ?

**Comment by cindychip** [comment]
Sorry about that. I just triggered the rerun. hopefully will resolve this issue.

**Pull request merged**